Rocket.Chat.ShareExtension/Rooms/SERoomsViewController.swift

```diff
@@ -61,6 +63,11 @@ extension SERoomsViewController: UISearchBarDelegate {
     }
 
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        guard !searchText.isEmpty else {
+            store.dispatch(.setSearchRooms(.none))
+            return
+        }
```

Comment:
@cardoso Opted to add this here, instead of in `SEStore`; because it improves readability...? 🤷🏻‍♂️

Reply:
This is fine too. :+1:

Reply:
This is fine too :+1: